vmc/resource_vmc_cluster.go

```diff
@@ -144,7 +144,7 @@ func resourceClusterRead(d *schema.ResourceData, m interface{}) error {
 
 	d.SetId(clusterID)
 	cluster := map[string]string{}
-	for i := 0; i < len(sddc.ResourceConfig.Clusters); i++ {
+	for i := 1; i < len(sddc.ResourceConfig.Clusters); i++ {
```

Comment:
why is the counter starting from 1 instead of 0?

Reply:
Done